dashboard/db/migrate/20190214005506_create_pd_scholarship_infos.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePdScholarshipInfos < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_scholarship_infos do |t|
+      t.belongs_to :user, foreign_key: true, null: false, index: true, dependent: :destroy
+      t.string :application_year, null: false
+      t.string :scholarship_status, null: false
+      t.belongs_to :pd_application, foreign_key: true, index: true
+      t.belongs_to :pd_enrollment, foreign_key: true, index: true
+      t.timestamps
+    end
```

Comment:
do these look like the right fields to index on?

Reply:
I don't know what kind of queries we want to optimize for but my hunch is indexing on user_id and enrollment_id seems right, application_id could be optional.

Reply:
I think we'll be looking these up by `application_id` to create/update them from the scholarship info on the detail view of an application, and by `enrollment_id` to create/update them from the workshop dashboard. I agree that indexing on `user_id` feels right but I don't know how we will actually use that yet.

Reply:
This one's a little tough - it also seems likely we'll want to report on, say, all scholarshipped teachers for a given year, but as more of a reporting-layer thing it may be less important for those queries to run very fast.  This looks good to me.